lib/x11utils.pm

```diff
@@ -67,11 +67,11 @@ all possible options should be handled within loop to get unlocked desktop
 
 =cut
 sub ensure_unlocked_desktop {
-    my $counter = 10;
+    my $counter = 30;
     while ($counter--) {
         my @tags = qw(displaymanager displaymanager-password-prompt generic-desktop screenlock screenlock-password authentication-required-user-settings authentication-required-modify-system);
         push(@tags, 'blackscreen') if get_var("DESKTOP") =~ /minimalx/;    # Only xscreensaver has a blackscreen as screenlock
-        assert_screen \@tags, no_wait => 1;
+        check_screen \@tags, no_wait => 1;
```

Comment:
Sorry, I *really* do not want to go back to `check_screen` here, *especially* not without checking the return code. Btw, the default time for `check_screen` is 0.

We should really avoid the situation that the "the screen can be anything right" and I fully trust we can find a limited set of expected screens. Btw, I am still for disabling the screensaver and power save in the general case and just keep it enabled for an explicit "screensaver/powersave" test: https://progress.opensuse.org/issues/40490

Reply:
Disabling screensaver/screenlock will have no effect here since this is just a transitory screen (black screen with a small dash on top left corner).
Going back to `assert_screen` with a check on  the "black screen with a small dash on top left corner" to wait more time would be ok for you?

Reply:
Yes, would be ok if we need to do something about it and if it is just waiting longer. You could push the tag only where needed. I don't know if that would be aarch64?

Did you read my remark regarding that I think we actually switch to the wrong tty in the first place?

Reply:
AFAICS, x86 is also affected.

I missed your comment about the wrong tty, but that would make sense with what I saw: `select_console(x11')` when we are already on `x11` should not give a `blackscreen-with-dash`. Any link?

Reply:
yes, https://progress.opensuse.org/issues/48110